src/lib/src/services/validators/validators/lowercase/lowercase.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ValidatorFn,
+  AbstractControl,
+  ValidationErrors,
+} from '@angular/forms';
+import { createContainsLowercaseRegex } from '@terminus/ngx-tools/regex';
+
+
+/**
+ * Return a validator function to verify that number of lowercase letters is satisfied
+ *
+ * @return The validator function
+ */
+export function lowercaseValidator(digits?: number): ValidatorFn {
+  return (control: AbstractControl): ValidationErrors | null => {
+    // Allow optional controls by not validating empty values
+    if (!control || !control.value) {
+      return null;
+    }
+
+    digits = digits ? digits : 1
+    const invalidResponse: ValidationErrors = {
+      lowercase: {
+        valid: false,
+        actual: control.value,
+        contains: digits,
```

Comment:
To match existing validator objects, we have been using the validator name as they object key that contains the 'rule' or 'limitation'.

Something like:

```javascript
{
  valid: false,
  lowercase: lowercaseMin,
  actual: control.value,
}
```